views/components/Matrix.d3.js

```diff
@@ -1,454 +1,599 @@
-import * as d3 from 'd3';
-import AbstractGraph from './AbstractGraph.d3';
-import { getColor } from '../../server/utils/invertColor';
-import { stringToDate } from '../../server/utils/dateConverter';
-import basePathConfig from '../../server/configs/basePathConfig';
+import * as d3 from 'd3'
 
-const basePath = basePathConfig || '';
+import { getColor } from '../../server/utils/invertColor'
+import { stringToDate } from '../../server/utils/dateConverter'
+import basePathConfig from '../../server/configs/basePathConfig'
+
+const basePath = basePathConfig || ''
```

Comment:
It looks like basePathConfig already [returns an empty string](https://github.com/AMP-SCZ/dpdash/blob/pnl-devel/server/configs/basePathConfig.js)?

Reply:
yeah good call. There's still a ton of refactor opportunity in this class. Trying not to get too carried away but this is a change I can make.